ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml

```diff
@@ -77,6 +77,13 @@ tests:
   commands: make test-unit
   container:
     from: src
+- as: golangci-lint
```

Comment:
why not just `lint`?

Reply:
to be explicit that this is running golancgi-lint :D

Reply:
https://github.com/openshift/release/pull/39954#discussion_r1214217382 :)

Reply:
but yeah,  we could have called the makefile target just `lint`. (this is just a detail, just to be consistent with other observability projects, you can ignore this)

Reply:
Heh, not really a strong opinion on that. I'd stick to golangci-lint to be explicit as I said but feel free to put up a PR to change this if you'd like it ;)